### PR TITLE
Deploy Worldchain testnet

### PR DESCRIPTION
**Pull request by bruce-riley**
This PR deploys World Chain to testnet and adds guardian support for it.

- The core is available [here](https://worldchain-sepolia.explorer.alchemy.com/address/0xe5E02cD12B6FcA153b0d7fF4bF55730AE7B3C93A).
- The token bridge is available [here](https://worldchain-sepolia.explorer.alchemy.com/address/0x430855B4D43b8AEB9D2B9869B74d58dda79C0dB2).

The change `ethereum/sh/registerAllChainsOnTokenBridge.sh` is to enabling registering the other chains on a new chain before support for this chain is added to sdk-v2 and the worm cli. It just removes an unnecessary check.

### CODE REVIEW

**Review by panoel — DISMISSED** [review]
I didn't verify the token bridge emitter address.

File: ethereum/env/.env.worldchain.testnet [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+INIT_EVM_CHAIN_ID=4801
+
+# Bridge Migrations
+BRIDGE_INIT_CHAIN_ID=45
+BRIDGE_INIT_GOV_CHAIN_ID=0x1
+BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+BRIDGE_INIT_FINALITY=1
+# I'm assuming this is right. . .
+BRIDGE_INIT_WETH=0x211Db8FbDc34982654e39b1B3a8Ca3EF5c7826EA
```
Comment (evan-gray) [inline ethereum/env/.env.worldchain.testnet:21]: as an OP Stack chain, shouldn't this be `0x4200000000000000000000000000000000000006`?

**Review by evan-gray — APPROVED** [review]
one comment regarding WETH config

**Review by evan-gray — APPROVED** [review]

**Review by gator-boi — APPROVED** [review]
I reviewed the `ethereum` directory.

**Pull request merged**